sass/_flag-icon-list.scss

```diff
@@ -46,6 +46,7 @@
   @include flag-icon(cf);
   @include flag-icon(cg);
   @include flag-icon(ch);
+  @include flag-icon(cht);
```

Comment:
You should move this down to the "Other flags" category as this is not a flag defined by ISO 3166.